docs/dg/dev/system-requirements/202407.0/system-requirements.md

```diff
@@ -0,0 +1,55 @@
+| OS                                        | Native: Linux                                                                                                                                                                                                                                                                                                                                                                                                  |
+| Web Server                                | NginX—preferred. But any webserver which supports PHP will work such as lighttpd, Apache, Cherokee.                                                                                                                                                                                                                                                                                                            |
+| Databases                                 | Depending on the project, one of the databases: MariaDB >= 10.4—preferred, PostgreSQL >=9.6, or MySQL >=5.7.                                                                                                                                                                                                                                                                                                   |
+| PHP                                       | Spryker supports PHP `>=8.1` with the following extensions: `curl`, `json`, `mysql`, `pdo-sqlite`, `sqlite3`, `gd`, `intl`, `mysqli`, `pgsql`, `ssh2`, `gmp`, `mcrypt`, `pdo-mysql`, `readline`, `twig`, `imagick`, `memcache`, `pdo-pgsql`, `redis`, `xml`, `bz2`, and `mbstring`. For details about the supported PHP versions, see [Supported Versions of PHP](/docs/dg/dev/supported-versions-of-php.html) |
+| SSL                                       | For production systems, a valid security certificate is required for HTTPS.                                                                                                                                                                                                                                                                                                                                    |
+| Redis                                     | Version >=5.0, >=6.2                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Where those versions came from?

Reply:
Here is the original documentation https://docs.spryker.com/docs/dg/dev/system-requirements/202311.0/system-requirements.html.

I also checked the supported versions for services here https://github.com/spryker/docker-sdk/tree/master/generator/src/templates/service